apps/web/vibes/soul/primitives/card-carousel/index.tsx

```diff
@@ -63,10 +77,12 @@ export function CardCarouselSkeleton({
   className,
   message,
   count = 8,
+  carouselScrollbarLabel,
```

Comment:
Rename to `scrollbarLabel`

Reply:
Also, it's weird we are passing this to the skeleton component. Could we refactor it so that this isn't required?

Reply:
Maybe let's create a `CarouselScrollbarSkeleton` component?

Reply:
I simply added a placeholder skeleton. We can refine in a different PR, maybe with help from @andrewreifman.